src/components/App.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import Motorcycles from './Motorcycles';
+import Layout from './layout/Layout';
+import Motorcycle from './motorcycles/Motorcycle';
+import MotorcycleForm from './motorcycleForm/MotorcycleForm';
+import Reservations from './Reservations';
+import ReserveForm from './reservationForm/ReservationForm';
+import MotorcycleDeleteForm from './motorcycleForm/MotorcycleDeleteForm';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this main `App` functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

- [ ] OPTIONAL: Refactor to ES6 the rest of the funcions.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/c00p75/appointment-app-frontend/assets/57421823/38738d66-7fc1-482a-bd9c-5d7786c9c20f)


That way, your code will meet the latest standards. 💪

Reply:
updated, thanks!